ff-api-update.php

```diff
@@ -1,32 +1,105 @@
 <?php
- 
+
 /* 
  * refresh the number of active nodes in the freifunk fulda community.
  * the node information is gathered from the ffmap nodes.json file and
  * written to the freifunk.net API file.
  */
- 
-$NODELIST = 'https://meshviewer.fulda.freifunk.net/data/nodes.json';
-$APIFILE = '/path/to/FreifunkFulda-api.json';
- 
-$map_json = file_get_contents($NODELIST);
-$map_php = json_decode($map_json);
- 
-$nodes = count((array)($map_php->{'nodes'}));
- 
+
+# Autoloader funktion (needed for 3rd-party validator)
+function __autoload($className)
+{
+    $className = ltrim($className, '\\');
+    $fileName  = '';
+    $namespace = '';
+    if ($lastNsPos = strrpos($className, '\\')) {
+        $namespace = substr($className, 0, $lastNsPos);
+        $className = substr($className, $lastNsPos + 1);
+        $fileName  = str_replace('\\', DIRECTORY_SEPARATOR, $namespace) . DIRECTORY_SEPARATOR;
+    }
+    $fileName .= str_replace('_', DIRECTORY_SEPARATOR, $className) . '.php';
+    if (stream_resolve_include_path($fileName)) {
+        require_once $fileName;
+    }
+}
+
+# disable SSL verification to prevent errors
+$arrContextOptions=array(
+    "ssl"=>array(
+        "verify_peer"=>false,
+        "verify_peer_name"=>false,
+    ),
+);
+
```

Comment:
without that options set I get the following errors/warnings:

```
`PHP Warning:  file_get_contents(): SSL operation failed with code 1. OpenSSL Error messages:
error:14090086:SSL routines:SSL3_GET_SERVER_CERTIFICATE:certificate verify failed in /home/christoph/freifunk/ff-api-updater/ff-api-update.php on line 92
PHP Warning:  file_get_contents(): Failed to enable crypto in /home/christoph/freifunk/ff-api-updater/ff-api-update.php on line 92
PHP Warning:  file_get_contents(https://meshviewer.fulda.freifunk.net/data/nodes.json): failed to open stream: operation failed in /home/christoph/freifunk/ff-api-updater/ff-api-update.php on line 92
```

I'm not sure how to solve this because the download of the API file from https://fulda.freifunk.net is working fine.